committing.md

```diff
@@ -207,9 +207,91 @@ Counting objects: 100% (1/1), done.
 Writing objects: 100% (1/1), 184 bytes | 184.00 KiB/s, done.
 Total 1 (delta 0), reused 0 (delta 0)
 To https://github.com/username/reponame
-   094eb39..cb3baec  main -> main
+   1f95680..5bf1f4e  main -> main
 ```
 
 Git will ask for your GitHub username and password.
 You won't see the password as you type it in, and that's completely normal.
 After pushing, you should immediately your changes on GitHub.
+
+
+## Looking at previous commits
+
+Run `git log` to get a list of all commits you have made so far.
+
+```sh
+$ git log
+commit 5bf1f4e2101b044e4032b23fe6940f3cd1c9f33f (HEAD -> main, origin/main, origin/HEAD)
+Author: yourusername <you@example.com>
+Date: Sun May 23 15:31:25 2021 +0300
+
+    add better description to README
+
+commit 1f956800248ad1d577039a92f05beee9e05c1a0f
+Author: yourusername <you@example.com>
+Date: Sun May 23 15:34:50 2021 +0300
+
+    Initial commit
+```
+
+As you can see, the latest commit is on top, and older commits are below it.
+If you have many commits, specify `--oneline` so you can fit more commits on the screen at once:
+
+```sh
+$ git log --oneline
+5bf1f4e (HEAD -> main, origin/main, origin/HEAD) add better description to README
+1f95680 Initial commit
+```
+
+Here `Initial commit` is what GitHub creates when you make a new repo,
+and "add better description to README" is the commit we just created.
+
+Each commit has a unique **commit hash**, sometimes also known as commit ID or SHA.
+For example, the hash of our latest commit is `5bf1f4e2101b044e4032b23fe6940f3cd1c9f33f`.
+The hashes are often abbreviated by taking just the first 7 characters,
```

Comment:
It can be more than 7 characters, if there's collisions within first 7 characters. For example, CPython has hashes like `68b9c20a98`.

```suggestion
The hashes are often abbreviated by taking a few characters from the beginning,
```